.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,7 +20,7 @@ For new packages please briefly describe the package or provide a link to its ho
   - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
   - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
   - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
-- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
+- [ ] Tested compilation of all packages that depend on this change using `nix run nixpkgs#nixpkgs-review rev HEAD` or `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
```

Comment:
Problem is however that this requires `--extra-experimental-features 'nix-command flakes'`

Reply:
I also don't see why `nix-shell` is worse in this case.

Reply:
Maybe I misunderstand the intended target audience here, I don’t know. If I can recite the “experimental features nix-command flakes” incantation in my sleep but the only thing I remember about `nixpkgs-review` is “there was this whatchamacallit mentioned in the PR template”, *each time*, am I in it? Perhaps not, if that wasn’t the intention then I’m fine with seeing this closed and moving on.